repository_service_tuf/cli/admin/sign.py

```diff
@@ -88,9 +167,25 @@ def sign(root_in, prev_root_in, save) -> None:
     signature = _add_signature_prompt(root_md, key)
 
     ###########################################################################
-    # Dump payload
-    # TODO: post to API
+    # Send payload to the API and save it locally
+    payload = SignPayload(signature=signature.to_dict())
     if save:
-        payload = SignPayload(signature=signature.to_dict())
-        json.dump(asdict(payload), save, indent=2)
+        json.dump(asdict(payload), save, indent=2)  # type: ignore
         console.print(f"Saved result to '{save.name}'")
+    elif signing_json_input_file:
+        with open("sign-payload.json", "w") as out_file:
+            json.dump(asdict(payload), out_file, indent=2)
+
+        console.print("Saved result to 'sign-payload.json'")
+
+    if not signing_json_input_file:
```

Comment:
This is not documented, and I'm not sure if it's intuitive. Shouldn't it be possible to also post a locally read and then signed payload to the API?